### PR TITLE
Update DPC++ fill test

### PR DESCRIPTION
**Pull request by rlnx**
Add checks for the fill result.

### CODE REVIEW

File: cpp/oneapi/dal/backend/primitives/test/fill_dpc.cpp [hunk 1]
```diff
@@ -18,26 +18,31 @@
 
 namespace oneapi::dal::backend::primitives::test {
 
-TEST("just fill", "[dpc++]") {
+TEST("fill device USM", "[usm]") {
```
Comment (michael-smirnov) [inline cpp/oneapi/dal/backend/primitives/test/fill_dpc.cpp:21]:
```suggestion
TEST("fill device USM and copy to host", "[usm]") {
```

**Review by KulikovNikita — APPROVED** [review]

**Review by michael-smirnov — APPROVED** [review]
LGTM
assuming private CI is working

**Pull request merged**